components/org.wso2.carbon.identity.application.authenticator.qrcode.servlet/src/main/java/org.wso2.carbon.identity.application.authenticator.qrcode.servlet/QRServletConstants.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.application.authenticator.qrcode.servlet;
+
+/**
+ * This class contains constants used for qr endpoint component.
+ */
+public class QRServletConstants {
+
+    public static final String QR_AUTH_ENDPOINT = "/qr-auth/authenticate";
+    public static final String QR_AUTH_STATUS_ENDPOINT = "/qr-auth/check-status";
+    public static final String SESSION_DATA_KEY = "sessionDataKey";
+    public static final String TENANT_DOMAIN = "tenantDomain";
+    public static final String AUTHENTICATED_USER = "authUser";
+    public static final String AUTH_DATA = "authData";
+    public static final String AUTH_STATUS = "authStatus";
+    public static final String AUTH_RESPONSE = "authResponse";
+    public static final String MEDIA_TYPE_JSON = "application/json";
+
+    /**
+     * Object holding authentication mobile response status.
+     */
+    public enum Status {
+        COMPLETED, PENDING
+    }
+
+    /**
+     * Enum which contains error codes and corresponding error messages.
+     */
+    public enum ErrorMessages {
+
+        ERROR_CODE_AUTH_RESPONSE_TOKEN_NOT_FOUND(
+                "PBA-15001",
```

Comment:
What is this "PBA" stands for? Better to use an abbreviation related to your component